code/modules/mapping/reader.dm

```diff
@@ -364,8 +364,8 @@
 
 	#ifdef TESTING
 	if(turfsSkipped)
-		testing("Skipped loading [turfsSkipped] default turfs")
 	#endif
+		testing("Skipped loading [turfsSkipped] default turfs")
```

Comment:
Why was this moved out of the testing ifdef

Reply:
funny how that didnt cause an error from the idnetation